### ISSUE TITLE
Hero title is not rendering in correct font in hero banner

### ISSUE THREAD

**Issue by Peter-holyoke** [ProvidenceGeeks/website-frontend#44]
## Type of Change
- [ ] New Feature Request
- [ ] Improvement / Suggestion
- [x] Bug
- [ ] Other (please clarify below)

## Summary
1. Hero Title in the hero banner does not render in the appropriate font, because the font file failed to load, due to a CORS problem.
1. macOS 10.12.6 - Google Chrome Version 61.0.3163.100 
1. Staging - http://stage.pvdgeeks.org/
1. Exact steps to reproduce
      1. Go to http://stage.pvdgeeks.org/
1. Expected result: The hero title should render in the Canter font family.
1. Actual result: The hero title renders in default font.
1. Error message:
```
Access to Font at 'http://d2b64jbw6hxz4f.cloudfront.net/bc517149950afc2999fdec6f2e1bf064.otf' from origin 'http://stage.pvdgeeks.org' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://stage.pvdgeeks.org' is therefore not allowed access.
```

**Comment by thescientist13** [comment]
The issue is due to `https` being used to include the external Google font
https://github.com/ProvidenceGeeks/website-frontend/blob/develop/src/components/bootstrap/bootstrap.scss#L8
```
@import url('https://fonts.googleapis.com/css?family=Josefin+Sans:300,400,400i,700|Josefin+Slab:300,400,400i,700');
```

But stage is only on HTTP.

Will try this trick of leaving out the scheme to allow the path to be HTTP / HTTPS agnostic, defaulting to whatever the current scheme is from where the asset was loaded from.
```
@import url('//fonts.googleapis.com/css?family=Josefin+Sans:300,400,400i,700|Josefin+Slab:300,400,400i,700');
```

**Comment by thescientist13** [comment]
looks like there still an error on stage :/
<img width="1427" alt="screen shot 2017-11-03 at 10 31 40 am" src="https://user-images.githubusercontent.com/895923/32378871-39f46ae0-c082-11e7-988e-8e281b099c1d.png">

**Comment by thescientist13** [comment]
Fixed for real this time @saimatsumoto !

The issue was that previous request for the HTTPS version of the font was cached, and so led to the CORS error persisting even after changing the font to only load over HTTP.  This required a simple cache invalidation of the font file ( _bc517149950afc2999fdec6f2e1bf064.otf_ ) in CloudFront.  
https://coderwall.com/p/ub8zug/serving-web-fonts-via-aws-s3-and-cloudfront

I have confirmed there is no error anymore
<img width="1184" alt="screen shot 2017-11-03 at 12 54 24 pm" src="https://user-images.githubusercontent.com/895923/32385985-2951fe8c-c096-11e7-9094-883e876703c0.png">

So in summary there were two issues
1.  The font was initially being loaded over HTTPS, when the Geeks site only support HTTPS
1.  Just fixing the path in the code didn't help since the external font file request from the client didn't change and so because it was already cached in CloudFront, we still got the old version.  So invalidating the cache for that specific path cleared up all our problems.

cc: @Peter-holyoke
